mysql-test/t/percona_bug1659992.test

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
No need for a new separate testcase - please merge with main.percona_userstat, replacing header comment with "--echo #" one